js/data.js

```diff
@@ -0,0 +1,125 @@
+const projects = [
+  {
+    id: 'project1',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+  {
+    id: 'project2',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+  {
+    id: 'project3',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+  {
+    id: 'project4',
+    title: 'Multi-Post Strories',
+    screenshot: './images/modal-image.png',
+    description: "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+    technologies: [
+      'css', 'html', 'bootstrap', 'ruby',
+    ],
+  },
+
+];
```

Comment:
- [ ] As it is a requirement for this project, which will also facilitate the adding of projects in the future, since all the information about the project will be added in one place, kindly add; 
-  link to live version,
- link to the project source code.
to your projects objects in the projects array. See screenshots of the requirement below for further clarifications.

![project-obj1](https://user-images.githubusercontent.com/62849872/151147843-98b5bb85-7507-441b-858a-4007bc0e1bbb.png)

![project-obj3](https://user-images.githubusercontent.com/62849872/151147915-8526e29a-067a-47d3-b55d-533d7d21e6a0.png)